packages/orchestration/src/examples/shared.flows.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * @import {Orchestrator, OrchestrationFlow, LocalAccountMethods} from '../types.js';
+ */
+
+// in guest file (the orchestration functions)
+// the second argument is all the endowments provided
```

Comment:
Is this comment necessary? Maybe you are envisioning some shared helpers that will rely on add'l endowments in the future?

A @file comment might be helpful here. Perhaps something to the effect of: 

Shared Flows are common async flow functions that can be reused across contracts. They are bound to a particular contract's context via `orchestrateAll`. See `./send-anywhere.contract.js` for example usage.